homeassistant/components/zeversolar/sensor.py

```diff
@@ -39,7 +39,7 @@ class ZeversolarEntityDescription(
 SENSOR_TYPES = (
     ZeversolarEntityDescription(
         key="pac",
-        icon="mdi:solar-power-variant",
+        translation_key="pac",
```

Comment:
Isn't this now changing the name of the entity?

Reply:
there is no corresponding key in the strings.json, so no